api/sirsidynix_authentication_provider.py

```diff
@@ -179,32 +162,24 @@ def remote_patron_lookup(
         if not data or "fields" not in data:
             return None
 
+        patrondata.complete = True
```

Comment:
Moved this up, so that `complete` always gets set, even if we return early from this function.